lms/djangoapps/courseware/views/views.py

```diff
@@ -1637,6 +1637,7 @@ def get_financial_aid_courses(user):
     for enrollment in CourseEnrollment.enrollments_for_user(user).order_by('-created'):
 
         if enrollment.mode != CourseMode.VERIFIED and \
+                enrollment.course_overview and \
```

Comment:
I see, maybe something about the structure of that particular course causes problems. Please add a comment to that effect here to aid future maintainers.